src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json

```diff
@@ -14,17 +14,15 @@
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
-
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "portable-net451+win8",
```

Comment:
Switching to full form to remove the odds of accidentally importing incorrect dependencies